### PR TITLE
Updated SDKs for bot framework and ...

### PR DESCRIPTION
**Pull request by krajniak**
Changed the BotProperties class so you can pass msaAppPassword to rest API

## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [ ] Please add REST spec PR link to the SDK PR
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/BotService/Management.BotService/Microsoft.Azure.Management.BotService.csproj [hunk 1]
```diff
@@ -50,6 +51,9 @@
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
   </ItemGroup>
+  <ItemGroup>
```
Comment (dsgouda) [inline src/SDKs/BotService/Management.BotService/Microsoft.Azure.Management.BotService.csproj]: We do not support Json 11.x.x at the moment. Please start an offline thread regarding your requirements here

**Review by dsgouda — CHANGES_REQUESTED** [review]

File: src/SDKs/BotService/Management.BotService/Microsoft.Azure.Management.BotService.csproj (hunk 1 above)
Comment (krajniak) [reply src/SDKs/BotService/Management.BotService/Microsoft.Azure.Management.BotService.csproj]: Can you set the proper nuget packages for this pull request?

Comment (dsgouda) [reply src/SDKs/BotService/Management.BotService/Microsoft.Azure.Management.BotService.csproj]: Nuget package references have been set globally [here](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/AzSdk.reference.props)
Hope that helps

Comment (krajniak) [reply src/SDKs/BotService/Management.BotService/Microsoft.Azure.Management.BotService.csproj]: Updated

Comment (dsgouda) [reply src/SDKs/BotService/Management.BotService/Microsoft.Azure.Management.BotService.csproj]: @krajniak could you tell us why you need these specific Newtonsoft references
Also, if you are trying to use a different auth mechanism to interact with the REST service, we do not support it at the moment.

**Comment by dsgouda** [comment]
@shahabhijeet PTAL

File: src/SDKs/BotService/Management.BotService/Microsoft.Azure.Management.BotService.csproj (hunk 1 above)
Comment (krajniak) [reply src/SDKs/BotService/Management.BotService/Microsoft.Azure.Management.BotService.csproj]: I don't, should I change it to not-specific version?

Comment (krajniak) [reply src/SDKs/BotService/Management.BotService/Microsoft.Azure.Management.BotService.csproj]: Ok, I've reverted the *.csproj changes, and only change the version of  Microsoft.Azure.Management.ResourceManager.Fluent to 1.16.0

**Comment by shahabhijeet** [comment]
@krajniak please use the above link to sign CLA before we even look at your contribution.

File: src/SDKs/BotService/Management.BotService/Microsoft.Azure.Management.BotService.csproj (hunk 1 above)
Comment (krajniak) [reply src/SDKs/BotService/Management.BotService/Microsoft.Azure.Management.BotService.csproj]: > @krajniak please use the above link to sign CLA before we even look at your contribution.

Done

**Comment by shahabhijeet** [comment]
@carlosscastro can you approve this PR

**Comment by krajniak** [comment]
Do anything more needs to be done to marge this pull request?

**Comment by shahabhijeet** [comment]
@krajniak sorry it's taking longer than expected. As the CI is incomplete, I failed to look at it.
I have added your user who can kick of the CI and it should kick off soon.
In the mean time @carlosscastro will be reviewing it soon.

**Comment by carlosscastro** [comment]
Thanks for your contribution @krajniak!!!

**Review by carlosscastro — APPROVED** [review]

**Comment by shahabhijeet** [comment]
@azuresdkci test this please

**Comment by krajniak** [comment]
Hello, any chance that new nuget package could be released.
@shahabhijeet @carlosscastro

**Comment by dsgouda** [comment]
@carlosscastro do you have plans to release a Nuget package. Let us know if you need help with it.

**Comment by dmaterowski** [comment]
kindly following up on this :) any ETA when we can expect a new package? @shahabhijeet  @carlosscastro

**Comment by dmaterowski** [comment]
@dsgouda @azuresdkci @shahabhijeet @carlosscastro It has been 53 days since PR was merged. 
Can anybody run the nuget release?

**Comment by dsgouda** [comment]
@carlosscastro do have any plans to release the SDK?

**Comment by dmaterowski** [comment]
@dsgouda what would need to happen for this release to be deployed?

**Comment by dsgouda** [comment]
@krajniak Please follow instructions [here](https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md) to kick off a publish job

**Comment by krajniak** [comment]
image

**Comment by krajniak** [comment]
@dsgouda

**Comment by dsgouda** [comment]
@krajniak unfortunately only SDK owners are allowed to actually publish the nuget.
@carlosscastro could you post some updates here

**Comment by dmaterowski** [comment]
@dsgouda since @carlosscastro doesn't seem to be available for the past two months, who else could publish the nuget?

**Comment by dsgouda** [comment]
We will try to find someone who can take a look at this. Thanks.

**Comment by dmaterowski** [comment]
:)

**Comment by dmaterowski** [comment]
@dsgouda any update on this?

**Comment by dsgouda** [comment]
@dmaterowski will follow up with the SDK owners again.

**Comment by dmaterowski** [comment]
@dsgouda who can I contact directly to get this resolved? stuck for over 82 days in build pipeline after merge.. that's not a very reasonable situation

**Comment by cosmaioan** [comment]
Do we have to build this manually in order to get a working version ? Does somebody use it like that ?

**Comment by dsgouda** [comment]
@carlosscastro Please open a new PR to check in [these](https://github.com/Azure/azure-sdk-for-net/pull/5356) changes and publish the package

**Pull request merged**